Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'rails-i18n', '~> 0.7.3'
 gem 'recaptcha', '~> 0.4.0', :require => 'recaptcha/rails'
 gem 'rmagick', '~> 2.14.0'
 gem 'ruby-msg', '~> 1.5.0',  :git => 'https://github.com/mysociety/ruby-msg.git', :ref => 'f9f928ed76c024b4bc3a08bc1a59beb62df36663'
+gem 'sass', '3.4.21'
```

Comment:
Ah

    Failure/Error: expect(r.out).to eq("")

       expected: ""
            got: "DEPRECATION WARNING:\nSass 3.5 will no longer support Ruby 1.9.3.\nPlease upgrade to Ruby 2.0.0 or greater as soon as possible.\n\n"

       (compared using ==)

       Diff:
       @@ -1 +1,4 @@
       +DEPRECATION WARNING:
       +Sass 3.5 will no longer support Ruby 1.9.3.
       +Please upgrade to Ruby 2.0.0 or greater as soon as possible.

     # ./spec/script/mailin_spec.rb:28:in `block (2 levels) in <top (required)>'

(spec run abandoned at that point)